src/library_promise.js

```diff
@@ -260,5 +260,20 @@ mergeInto(LibraryManager.library, {
     dbg(`create: ${id}`);
 #endif
     return id;
-  }
+  },
+
+#if ASYNCIFY == 2
+  emscripten_promise_await_sync__deps: ['$getPromise'],
+  emscripten_promise_await_sync__sig: 'pp',
```

Comment:
`__sig` attributes should no longer be needed.

`__async` attribute marks the functions as async